core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -1315,8 +1315,56 @@ public String getPerfLoggerClassName() {
         return getOptional("kylin.metrics.perflogger-class", "org.apache.kylin.common.metrics.perflog.PerfLogger");
     }
 
-    public String getMetricsActiveReservoirDefaultClass() {
-        return getOptional("kylin.metrics.active-reservoir-default-class",
+    public boolean isKylinMetricsMonitorEnabled() {
+        return Boolean.parseBoolean(getOptional("kylin.core.metrics.monitor-enabled", "false"));
+    }
+
+    public boolean isKylinMetricsReporterForQueryEnabled() {
+        return Boolean.parseBoolean(getOptional("kylin.core.metrics.reporter-query-enabled", "false"));
+    }
+
+    public boolean isKylinMetricsReporterForJobEnabled() {
+        return Boolean.parseBoolean(getOptional("kylin.core.metrics.reporter-job-enabled", "true"));
```

Comment:
Better to disable job metrics before system cube check-in?

Reply:
Agree